android/src/main/kotlin/com/adyen/checkout/flutter/CheckoutPlatformApi.kt

```diff
@@ -87,20 +88,21 @@ class CheckoutPlatformApi(
             }
 
             is CardComponentConfigurationDTO -> {
-                return configuration.cardConfiguration.toNativeModel(
+                return configuration.cardConfiguration.fromDTO(
```

Comment:
IMHO
`fromDTO` doesn't sound as good method name. Method suppose to start with verb. Exception here is `toXXX`, since it is short form of `convertToXXX`.
I can think of static method `createFrom(xxx)` that shortened to `from(xxx)`, but in case above naming seems confusing

Reply:
Sure, let's change it to a better method name.